omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -117,6 +117,17 @@
 default['private_chef']['rabbitmq']['env_path'] = "/opt/opscode/bin:/opt/opscode/embedded/bin:/usr/bin:/bin"
 
 ####
+# External RabbitMQ
+####
+default['private_chef']['external-rabbitmq']['enable'] = false
```

Comment:
That is a good point - although I think it might be easier for the users to flip their external rabbitmq on and of with a single config variable rather than having to remove/rewrite all variables into the chef-server.rb